output/json/json.go

```diff
@@ -171,6 +170,29 @@ func (o *Output) handleMetric(m *metrics.Metric, jw *jwriter.Writer) {
 	}
 	o.seenMetrics[m.Name] = struct{}{}
 
-	wrapMetric(m).MarshalEasyJSON(jw)
+	//nolint:exhaustruct
+	wrapped := metricEnvelope{
+		Type:   "Metric",
+		Metric: m.Name,
+	}
+	wrapped.Data.Name = m.Name
+	wrapped.Data.Type = m.Type
+	wrapped.Data.Contains = m.Contains
+
+	if ts, ok := o.thresholds[m.Name]; ok {
+		wrapped.Data.Thresholds = ts
+	}
+
+	// TODO: refactor after the other refactors
+	// in the metrics area will be completed.
+	//
+	// This parts can be racy because
+	// they are controlled and written from the metrics.Engine.
```

Comment:
Seems to me that Submetrics isn't really racy as they get initialized early (after some of the previous refactoring). 

And Tainted is in practice always going to be null at this time as thresholds get ran on a timer, and it's unlikely that timer has ran at the first time the JSON output will be outputting.  So maybe we should just drop `Tainted` as a slight breaking change. From what i can see again it would've always been null, so it seems not all that relevant :shrug:

Reply:
How the output is implemented at the moment it doesn't make sense to be printed. We are just flushing the metrics the first time and we skip all the other times. So it seems reasonable to me to remove it because the value will be updated probably after the first time.

@na-- WDYT? Is it the right moment for this? Do we have future plans for the JSON output where `tainted` could be more valuable?

Reply:
For the record, consensus to drop `tainted` achieved privately